tests/utils/reporting.py

```diff
@@ -23,7 +23,7 @@
 # A mapping to fix the mismatch between the debug_info and the identifiers.
 _label_scope = {
     "kakarot.constants.opcodes_label": "kakarot.constants",
-    "kakarot.accounts.contract.library.internal.pow_": "kakarot.accounts.contract.library.internal",
+    "kakarot.accounts.library.internal.pow_": "kakarot.accounts.library.internal",
```

Comment:
have you just check with a `--profile-cairo` that it was working?

Reply:
<img width="811" alt="image" src="https://github.com/kkrt-labs/kakarot/assets/60658558/8d84efdc-ca9b-400e-985e-ddf824fbb431">

Looks like yes